src/chttpd/src/chttpd_auth_cache.erl

```diff
@@ -84,11 +83,20 @@ get_from_cache(UserName) ->
             end
     catch
         error:badarg ->
-            couch_stats:increment_counter([couchdb, auth_cache_misses]),
-            couch_log:debug("cache miss for ~s", [UserName]),
+            maybe_increment_auth_cache_miss(UserName),
             load_user_from_db(UserName)
     end.
 
+maybe_increment_auth_cache_miss(UserName) ->
+    Admins = config:get("admins"),
+    case proplists:get_value(?b2l(UserName), Admins) of
```

Comment:
If all we want is to check if user is in admin list `lists:member/2` works better here. It's faster too.

Reply:
@eiri good observation, however, `config:get(section)` returns a proplist like `[{Name, PassHash}],{Name2,Pass2Hash}]`, so it’d have to be at least `lists:keymember()`, but I’m not sure it is faster.

Reply:
Looks like it is: https://stackoverflow.com/a/30812279

Reply:
It is, actually, `lists` operations are `nifs` and `proplists` are pure erlang module. Plus `keymember` still better semantically.

Reply:
fixed, h/t